### PR TITLE
Create Require_MFA_VAgov.md

### PR DESCRIPTION
**Pull request by joeniquette**
PR which adds a policy for requiring verified mfa accounts for dd changes

### CODE REVIEW

File: products/identity/Policies and Procedures/Require_MFA_VAgov.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# MFA Required on VA.gov
+
+### Version History
+| Version Number | Author(s) | Revision Date | Description of Change |
+| --- | --- | --- | --- |
+| 1.0 | Joe Niquette | 5/19/23 | Initial creation, approval provided via email to the VA.gov Identity team by Cody Reinold. This document formally acknowledges that approval by the PR approval of Thomas Black. |
+
+## Background
+
+Fraudulent Direct Deposit account changes on VA.gov are on the rise at VA. In an attempt to reduce fraudulent activity on VA.gov the VA.gov Identity team has implemented a policy that requires all direct deposit changes be made by an MFA (multi-factor authentication) required verified ID.me or Login.gov account. This policy was set by the Identity teams PO (Product Owner) at the time the policy was set, Cody Reinold. The policy was set and is not set to expire effective 9/20/2021.
```
Comment (asg5704) [inline products/identity/Policies and Procedures/Require_MFA_VAgov.md:10]: > Fraudulent Direct Deposit account changes on VA.gov are on the rise at VA. In an attempt to reduce fraudulent activity on VA.gov the VA.gov Identity team has implemented a policy that requires all direct deposit changes be made by an MFA (multi-factor authentication) required verified ID.me or Login.gov account. This policy was set by the Identity teams PO (Product Owner) at the time the policy was set, Cody Reinold. The policy was set and is not set to expire effective 9/20/2021.

Fraudulent direct deposit account changes on VA.gov are on the rise at the Department of Veterans Affairs. In an attempt to reduce fraudulent activity on VA.gov, the VA.gov Identity team has implemented a policy that requires all direct deposit changes to be made with a verified ID.me or Login.gov account that requires MFA (multi-factor authentication). This policy was set by the Identity team's PO (Product Owner) at the time, Cody Reinold. The policy has been set and is not set to expire until 9/20/2021.

File: products/identity/Policies and Procedures/Require_MFA_VAgov.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Policy
+
+All bank account (direct deposit) detail modifications within the VA.gov domain must utilize a multi-factor required verified ID.me or Login.gov account. Services which provide the option to modify bank account details must also confirm multi-factor was successfully entered by the current user during the session in which the account changes are being requested.
```
Comment (asg5704) [inline products/identity/Policies and Procedures/Require_MFA_VAgov.md]: > utilize a multi-factor required verified ID.me or Login.gov account
utilize verified ID.me or Login.gov accounts with required multi-factor authentication

File: products/identity/Policies and Procedures/Require_MFA_VAgov.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+```elixir
+data.attributes.profile.multifactor=true AND 
+data.attributes.profile.loa.current=3 AND
+data.attributes.profile.signIn.serviceName=(logingov OR idme)
+```
+
+If these conditions are not met then the service must not allow for the submission or request to be forwarded to the downstream service which consumes the bank account changes.
+
+In order to ensure continued access to modify bank account details users which do not currently meet the requirements of a multi-factor required verified ID.me or Login.gov account, the service must display content which explains why the current account does not meet the requirements of VA. The service must also present the user with a path to create either of the currently approved multi-factor required verified ID.me or Login.gov accounts.
```
Comment (asg5704) [inline products/identity/Policies and Procedures/Require_MFA_VAgov.md:28]: Replace with
In order to ensure continued access to modify bank account details, users who do not currently meet the requirements: 1.) a verified ID.me or Login.gov account AND 2.) multi-factor authentication enabled, must be shown content that explains why their current account does not meet the requirements of VA. The service must also provide the user with a path to meet the VA’s requirements by creating a verified account with ID.me or Login.gov accounts AND setting up the verified account with multi-factor authentication.

**Review by asg5704 — CHANGES_REQUESTED** [review]
Comments left

File: products/identity/Policies and Procedures/Require_MFA_VAgov.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Policy
+
+All bank account (direct deposit) detail modifications within the VA.gov domain must utilize a verified ID.me or Login.gov account with required multi-factor authentication. Services which provide the option to modify bank account details must also confirm multi-factor was successfully entered by the current user during the session in which the account changes are being requested. Modifications on a bank account must be performed by a user that is directly authenticated, modifications cannot be made on behalf of another user.
+
+## Execution
+
+Services within VA.gov must validate the current users details from the `user model`. The fields from the `user model` which must be present at the time the modifications are being requested are:
+
+```elixir
```
Comment (bosawt) [inline products/identity/Policies and Procedures/Require_MFA_VAgov.md:20]: elixir?????

**Review by asg5704 — APPROVED** [review]

**Review by blacktm — APPROVED** [review]

**Pull request merged**